Formula/konoha.rb

```diff
@@ -3,6 +3,7 @@ class Konoha < Formula
   homepage "https://github.com/konoha-project/konoha3"
   url "https://github.com/konoha-project/konoha3/archive/v0.1.tar.gz"
   sha256 "e7d222808029515fe229b0ce1c4e84d0a35b59fce8603124a8df1aeba06114d3"
+  revision 1
```

Comment:
@DomT4 Is this change what you had in mind? I looked at the Jenkins errors, and they don't make sense to me because `brew install konoha` works for me, `libmpi.20.dylib` replaces 'libmpi.12.dylib`, and `konoha` doesn't use the C++ MPI bindings, so `libmpi_cxx_1.dylib` should not be needed.